Make.config

```diff
@@ -126,9 +126,9 @@ MACCATALYST_NUGET_VERSION_NO_METADATA=$(MACCATALYST_NUGET_VERSION)-$(NUGET_PRERE
 MACCATALYST_NUGET_VERSION_FULL=$(MACCATALYST_NUGET_VERSION_NO_METADATA)+$(NUGET_BUILD_METADATA)
 
 # Xcode version should have both a major and a minor version (even if the minor version is 0)
-XCODE_VERSION=12.5
-XCODE_URL=https://dl.internalx.com/internal-files/xcodes/Xcode_12.5.xip
-XCODE_DEVELOPER_ROOT=/Applications/Xcode_12.5.0.app/Contents/Developer
+XCODE_VERSION=13.0
+XCODE_URL=https://dl.internalx.com/internal-files/xcodes/Xcode_13.0-beta.xip
```

Comment:
Internal email provided this URL `https://bosstoragemirror.azureedge.net/internal-files/xcodes/Xcode_13_beta.xip`
and it seems to work fine

Reply:
well `dl.internalx.com` should work too - but the filename is different, you have an extra `.0`

Reply:
Fixed too.